datapower-docker-compose-dp-mq/src/mq/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM mq:8
+ENV LICENSE=accept \
+    MQ_QMGR_NAME=TEST_GROW_QM_REQ
+COPY config.mqsc /etc/mqm/
+RUN useradd joedoe -G mqm && \
```

Comment:
Maybe make a comment about how it is not desired to include admin passwords in this way?